service/src/test/java/com/derpgroup/derpwizard/AppTest.java

```diff
@@ -61,6 +62,6 @@ public void runSuccess() throws Exception {
 
     // Verify that health checks and resources are correctly registered
     verify(mockHealthChecks).register(Matchers.matches("basics"), Matchers.any(BasicHealthCheck.class));
-    verify(mockJersey).register(Matchers.any(HomeResource.class));
+    verify(mockJersey, times(2)).register(Matchers.any(HomeResource.class));
```

Comment:
We should fix this so that it checks for the HomeResource and the AlexaResource.